src/hotspot/cpu/riscv/vm_version_riscv.hpp

```diff
@@ -39,21 +39,28 @@ class VM_Version : public Abstract_VM_Version {
 
   constexpr static bool supports_stack_watermark_barrier() { return true; }
 
-  static bool is_checkvext_fault(address pc) {
-    return pc != NULL && pc == _checkvext_fault_pc;
-  }
 
-  static address continuation_for_checkvext_fault(address pc) {
-    assert(_checkvext_continuation_pc != NULL, "not initialized");
-    return _checkvext_continuation_pc;
-  }
+  enum Feature_Flag {
+#define CPU_FEATURE_FLAGS(decl)               \
```

Comment:
Please keep those macros in the same order as the HWCAP_ISA_X macros in file vm_version_linux_riscv.cpp.

Reply:
fixed